client/my-sites/comments/index.js

```diff
@@ -18,14 +34,23 @@ export default function () {
 	page(
 		'/comments/:status(all|pending|approved|spam|trash)/:site/:post',
 		siteSelection,
+		redirectToCommentIfDuplicatedView( 'edit-comments.php' ),
 		navigation,
 		postComments,
 		makeLayout,
 		clientRender
 	);
 
 	// Comment View
-	page( '/comment/:site/:comment', siteSelection, navigation, comment, makeLayout, clientRender );
+	page(
+		'/comment/:site/:comment',
+		siteSelection,
+		redirectToCommentIfDuplicatedView( 'comment.php?action=editcomment' ),
```

Comment:
Technically, the comment view in Calypso is not equivalent to the edit view in WP Admin:

Comment view (Calypso) | Edit view (WP Admin)
--- | ---
<img width="974" alt="Screenshot 2024-12-18 at 12 19 40" src="https://github.com/user-attachments/assets/776e2280-2154-4213-bfd0-af19601cc2e4" /> | <img width="988" alt="Screenshot 2024-12-18 at 12 20 19" src="https://github.com/user-attachments/assets/6d4e1a66-a50c-4c04-aab3-3621c6f16ca5" />

But I don't think we have anything similar in WP Admin, so probably the edit view is our best option.

Reply:
Yeah, I'm aware. But it's our best bet at the moment.